src/mixins/filing-template-mixin.ts

```diff
@@ -363,12 +367,6 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         break
     }
 
-    // If this is a named IA then add Name Request Number and Approved Name.
-    if (this.isNamedBusiness) {
-      filing.registration.nameRequest.nrNumber = this.getNameRequestNumber
-      filing.registration.nameRequest.legalName = this.getApprovedName
-    }
```

Comment:
I believe this block is obsolete, since the draft registration filing already contains a nameRequest object with the 3 fields that we care about. @cameron-freshworks , thoughts?

The app still fetches the actual NR and then validates it, but the app NR data is not used in the registration filing object.

This feels a bit awkward but there's a reason for both to exist (outside the control of this app) so it is what it is.